Common/StreamWrapper/tests/Common/TestPayload.cs

```diff
@@ -40,11 +40,11 @@ public enum TaskType
       DatadepCompute,
     }
 
-    public byte [] DataBytes { get; set; }
+    public byte[]? DataBytes { get; set; }
```

Comment:
Do you really need to distinguish between null and empty? couldn't empty be enough to encode a missing data?

Reply:
In the property you're right but in the test there is warning to avoid null object

Reply:
ok